corehq/apps/enterprise/tests/test_interface.py

```diff
@@ -245,6 +245,8 @@ def create_smsbillable(self, date, show_billable=True, gateway_fee=None):
         else:
             smsbillable.gateway_fee = None
         smsbillable.save()
+        SmsBillable.objects.filter(id=smsbillable.id).update(date_created=date)
```

Comment:
How come we do this here, instead of on line 238 when we create `smsbillable`?

Reply:
no specific reason